sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobDownloadTest.groovy

```diff
@@ -0,0 +1,656 @@
+package com.azure.storage.blob.specialized.cryptography
```

Comment:
I'd prefer to not have test files specific to methods. Download tests should be in the file that tests other methods on EncryptedBlob

Reply:
I'll move em